session.go

```diff
@@ -832,25 +854,13 @@ func (s *session) loadCommonGlobalVariablesIfNeeded() error {
 	}
 	// Set the variable to true to prevent cyclic recursive call.
 	vars.CommonGlobalLoaded = true
-	save := privilege.GetPrivilegeChecker(s)
```

Comment:
Because now `ExecRestrictedSQL` use special session and don't check privilege , so this is not need any more.